src/test/java/org/commcare/backend/suite/model/test/CaseClaimModelTests.java

```diff
@@ -112,6 +120,23 @@ private void testGetRawQueryParamsWithUserInput(Map<String, String> userInput, L
         Assert.assertEquals(expected, params.get("_xpath_query"));
     }
 
+    private void testGetRawQueryParamsWithUserInputExcluded(Map<String, String> userInput, List<String> expected)
+            throws Exception {
+        MockApp mApp = new MockApp("/case_claim_example/");
+
+        SessionWrapper session = mApp.getSession();
+        session.setCommand("patient-search");
+
+        RemoteQuerySessionManager remoteQuerySessionManager = RemoteQuerySessionManager.buildQuerySessionManager(
+                session, session.getEvaluationContext(), new ArrayList<>());
+
+        userInput.forEach(remoteQuerySessionManager::answerUserPrompt);
+
+        Multimap<String, String> params = remoteQuerySessionManager.getRawQueryParams(false);
+
+        Assert.assertEquals(expected, params.get("_xpath_query_exclude"));
```

Comment:
This should be looking at the `exclude_patient_id` param. Also I would expect the param to be missing entirely rather than have an empty value:

```
Assert.assertFalse(params.containsKey("exclude_patient_id"))
```